backend/orchestrator/resources/dynamodb.ts

```diff
@@ -14,7 +14,9 @@ export class OrchestratorDynamodb extends ServerlessCdkPlugin.ServerlessConstruc
   constructor(scope: Construct, id: string, serverlessProps: ServerlessProps) {
     super(scope, id, serverlessProps);
 
-    const table = new Table(this, 'OrchestratorTable', {
+    const table = new Table(this, `OrchestratorTable`, {
+      // tableName: `table-${serverlessProps.serverless.provider.stage ?? ''}`,
+      tableName: `table-${serverlessProps.service.provider.stage}`,
```

Comment:
Setting the resource names is not a good practive IMO, do you think you could demontrate this capability with another setting?

Reply:
For example, a worthy demonstration would be to set the a different deletion policy depending on the stage, WDYT?

Reply:
Yes this was just to test things out, and I could not get the actions to run with the PR set as draft. I gave an example in the docs now :)

Reply:
I included your excellent idea in the example as well, thanks !